R/constants.R

```diff
@@ -45,9 +54,12 @@ CONV_BIL_MIL <- 1000
 CONV_MIL_THOUS <- 1000
 CONV_ONES_THOUS <- 0.001
 CONV_TON_MEGATON <- 1e-6
+CONV_T_KG <- 1e3
+CONV_T_METRIC_SHORT <- 1000/908
```

Comment:
Can we get a comment here explaining what this division is? I assume ratio between metric ton and short ton?